helm/values/prod.yaml

```
@@ -0,0 +1,5 @@
+ingress:
```

```suggestion
extraLabels:
  env: prod
ingress:
```

We might as well match internal convention.